### ISSUE TITLE
Add conversion to 960x720

### ISSUE THREAD

**Issue by anonymlol** [amichaelt/BDSup2SubPlusPlus#30]
Could you please make it possible to convert the resolution to 960x720?

**Comment by ghost** [comment]
Yes, the ability to specify an a custom resolution to convert to would be a good thing to add.  Won't make it in until at least the next version, though.
